_docs/manage/users.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: guide
+title: StorageOS Docs - Users
+anchor: manage
+module: manage/users
+---
+
+# Users
+
+User accounts facilitate granular permissioning of acceptable actions within the system by the use of policies.
+
+>**Note**: On initial startup there is only the storageos user which is an admin user.
+
+## Admin vs User accounts
+
+Admin users are treated as super-users which can perform any action within the system, regardless of policies set.
+Admins also have the added ability to add, update and remove users policies and perform other administrative roles within the system.
+
+Users only have the ability to change their password and have access to the namespaces granted to them by the policies set.
+
+## CLI
+### Creating a new user
+
+Users can be created, deleted and managed from the storageos CLI. For example to create a new non-admin user 'awesomeUser' and add them to the 'test' and 'dev' groups, Run:
+
+```bash
+$ storageos user create bob --groups test,dev --password --role user awesomeUser
+Password: 
+Confirm Password: 
+```
+
+>**Note**: Passwords are prompted for interactively to avoid passwords appearing in terminal logs
+
+### Listing current users
```

Comment:
(Throughout) Prefer "List current users", "Remove users" over "Listing" and "Removing".